imports/plugins/core/ui/client/components/forms/form.js

```diff
@@ -1,22 +1,41 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { map, update, set, at, isEqual } from "lodash";
+import { at, get, isEqual, map, set, update, extend } from "lodash";
```

Comment:
Is there any need for `map` or `extend` in an ES6 world?

Reply:
Not really, just that I thought it would look consistent if all object/array functions are used from the same package and have similar syntax while being used.
Though not very attached to this, let me know and I will change it.

Reply:
Yes, you are correct that we should be consistent and also follow the style guide. So we want to eliminate usages of `lodash` whenever there is an ES6-native equivalent.
https://docs.reactioncommerce.com/reaction-docs/master/styleguide